### DISCUSSION TITLE
Stream Support

### DISCUSSION THREAD

**Discussion by jeffective** [thorgrimjansrud/redis-codesys-lib#2] [Ideas]
Thank you for sharing this library!!!

It seems to me like the ideal communication could be using redis streams (XADD, XREAD, etc). This could facilitate extremely fast data collection scenarios (1000s of tags at 1000hz). What do you think about redis stream support?

**Comment by thorgrimjansrud** [comment]
Thanks for the feedback, it would sure be interesting to see the limitations in the codesys-runtime! To early to say anything spesific at this point :-).
